pkg/manager/member/pd_upgrader.go

```diff
@@ -115,7 +115,7 @@ func (pu *pdUpgrader) upgradePDPod(tc *v1alpha1.TidbCluster, ordinal int32, newS
 		lastOrdinal := tc.PDStsActualReplicas() - 1
```

Comment:
The `lastOrdianl` here should also be calculated by `GetMaxPodOrdinal`, right?

Reply:
I agree with you.Thanks.